server/controllers/student/getStudent.js

```diff
@@ -1,18 +1,18 @@
-const getStudentInformation = require('../../../database/queries/getStudent');
+const getStudentInformation = require('../../database/queries/getStudent');
 
-exports.getUserInformation = (req, res) => {
+exports.getStudent = (req, res) => {
   const { id } = req.params;
   getStudentInformation(id)
-    .then((information) => {
-      if (!information.rowCount) {
+    .then((student) => {
+      if (!student.rowCount) {
         res.status(404).send({
           error: 'The student dose not exist',
           data: null,
         });
       } else {
         res.send({
           error: null,
-          data: information.rows,
+          data: student.rows,
```

Comment:
@AbdallahAmmar96 Here also you could return `student.rows[0]` just to make the data a little easier for the client to consume.

Reply:
Done